godtools/App/Share/SwiftUI Views/AnimatedSwiftUIView/AnimatedSwiftUIView.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  AnimatedSwiftUIView.swift
+//  godtools
+//
+//  Created by Levi Eggert on 6/10/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+import Lottie
+import SwiftUI
+
+struct AnimatedSwiftUIView: UIViewRepresentable {
```

Comment:
Was this working for you?  I wasn't seeing it animate-- Honor Restored should be one that does, right?

Reply:
Hey @rachaelblue just checked this in Staging and animations are playing, but the frame isn't resizing to fit.  Going to fix this in this PR.